index.html

```diff
@@ -68,7 +68,78 @@ <h3>Multi-Post Stories</h3>
                 <div class="item3">Ruby</div>   
             </div>
-            <button>Select Project</button>
+            <button type="button" class="work-button">Select Project</button>
+            <div class="job1">
+                <h2>Profesional Art Printing Data</h2>
+                <div class="info-data"><p>A daily section of daily private personalised reads no accounts or sign ups required ha been the industry standups</p></div>
+                <ul class="grid-container2">
+                    <li class="item1"><a href="#">html</a></li>
+                    <li class="item2"><a href="#">bootstrap</a></li>
+                    <li class="item3"><a href="#">Ruby</a></li>  
+                </ul>
+                <button type="button" class="job-button">See Project</button>
+            </div>
+            <div class="job1">
+                <h2>Profesional Art Printing Data</h2>
+                <div class="info-data"><p>A daily section of daily private personalised reads; no accounts or sign ups required ha been the industry standups</p></div>
+                <ul class="grid-container2">
+                    <li class="item1"><a href="#">html</a></li>
+                    <li class="item2"><a href="#">bootstrap</a></li>
+                    <li class="item3"><a href="#">Ruby</a></li>  
+                </ul>
+                <button type="button" class="job-button">See Project</button>
+            </div>
+            <div class="job1">
+                <h2>Profesional Art Printing Data</h2>
+                <div class="info-data"><p>A daily section of daily private personalised reads; no accounts or sign ups required ha been the industry standups</p></div>
+                <ul class="grid-container2">
+                    <li class="item1"><a href="#">html</a></li>
+                    <li class="item2"><a href="#">bootstrap</a></li>
+                    <li class="item3"><a href="#">Ruby</a></li>  
+                </ul>
+                <button type="button" class="job-button">See Project</button>
+            </div>
+        </section>
+        <section>
+            <div class="me">
+                <div class="about-me">
+                    <h1>About me</h1>
+                </div>
+                <p class="about-info">Hi I am a software developer I can help you build a product,feature or website.Look through some of my work and experience! if you like what you see and have a project you need coded dont hesitate to contact me</p>
+                <button type="button" class="about-button">Get My Resume</button>
+                <div class="deco">
+                    <img class="wall" src="/assets/vectors/Vector-3.png" alt=""/>
+                    <img class="try" src="/assets/vectors/Vector-2.png" alt=""/>
+                    <img class="vector4" src="/assets/vectors/Vector-4.png" alt=""/>
+                    <img class="straight" src="/assets/vectors/Vectorin4.png" alt=""/>
+                </div>
```

Comment:
Please use `<ul>` `<li>` tags for displaying list of items. Kindly replace `<div>` with the mentioned tags. 😄

Reply:
Hi Zainsadaqat. those are images for decoration in the file not a list in anyway